test/test_functions

```diff
@@ -1181,3 +1188,125 @@ cp_usrbin() {
     sudo chmod 0444 $f || return 3
   done
 }
+
+
+# waits until cvmfs is mounted
+wait_fqrn_mount() {
+  local waits=0
+  while ! mount | grep "/cvmfs/${FQRN}" > /dev/null 2>&1; do
+    waits=$(( waits + 1 ))
+    if [ "$waits" -ge 10 ]; then
+      die "${FQRN} could not be mounted (not superuser?)"
+    fi
+    sleep 1
+  done
+}
+
+
+# runs the command once in order to get a warm cache
+create_warm_cache() {
+  local workspace=$1
+  cvmfs2 -f -o config=cvmfs.conf,disable_watchdog,simple_options_parsing $FQRN /cvmfs/$FQRN &> /dev/null &
+  wait_fqrn_mount
+
+  bash -c "cvmfs_run_benchmark" > /dev/null 2>&1
```

Comment:
We probably do want to keep the stdout and stderr messages.  Have a look in the integration tests, at some point the output of the testcases is redirected into the logfile specified as a parameter to `run.sh`.  We'd probably like to have the same behavior for the benchmarks.